gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -31,6 +31,12 @@ from google.auth.transport.grpc import SslCredentials             # type: ignore
 from google.auth.exceptions import MutualTLSChannelError          # type: ignore
 from google.oauth2 import service_account                         # type: ignore
 
+try:
```

Comment:
suggest macro; see other comment

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/a5d6109fb82fb2770afd54267d023c30484eaa0d and https://github.com/googleapis/gapic-generator-python/pull/1999/commits/109edd462b49315450ca2d6672dc408069834bdc